docs/source/user/rtc.rst

```diff
@@ -60,3 +60,22 @@ on the same document.
     looses connection and goes offline for a while. You should never have to touch it, and it is
     fine to just ignore it, including in your version control system (don't commit this file). If
     you happen to delete it, there shouldn't be any serious consequence either.
+
+There are a number of settings that you can change:
+
+.. code-block:: bash
+
+  $ # The delay of inactivity (in seconds) after which a document is saved to disk (default: 1).
+  $ # If None, the document will never be saved.
+  $ jupyter lab --collaborative --YDocExtension.document_save_delay=0.5
+  $
+  $ # The period (in seconds) to check for file changes on disk (default: 1).
+  $ # If 0, file changes will only be checked when saving.
+  $ jupyter lab --collaborative --YDocExtension.file_poll_interval=2
+  $
+  $ # The delay (in seconds) to keep a document in memory in the back-end after all clients disconnect (default: 60).
+  $ # If None, the document will be kept in memory forever.
+  $ jupyter lab --collaborative --YDocExtension.document_cleanup_delay=100
+  $
+  $ # The YStore class to use for storing Y updates (default: JupyterSQLiteYStore).
+  $ jupyter lab --collaborative --YDocExtension.ystore_class=ypy_websocket.ystore.TempFileYStore
```

Comment:
These extra `$` look a bit odd on the RTD doc preview: https://jupyterlab--13413.org.readthedocs.build/en/13413/user/rtc.html

![image](https://user-images.githubusercontent.com/591645/201971808-78b8c9ca-bc67-4f43-b846-a5d0207891cc.png)

Maybe they can removed, or only keep the ones in front of the actual commands (not the comments)?

Reply:
I took [this page](https://jupyterlab.readthedocs.io/en/latest/user/urls.html#managing-workspaces-cli) as a model, but I'm fine with removing the `$`.

Reply:
ah ok thanks for linking to that page :+1: